src/FuturesClientV2.ts

```diff
@@ -266,6 +266,12 @@ export class FuturesClientV2 extends BaseRestClient {
     return this.postPrivate('contract/private/cancel-orders', params);
   }
 
+  cancelAllFuturesOrderTimed(params: {
```

Comment:
```suggestion
  cancelAllFuturesOrdersAfterTimeout(params: {
```

I know it's lengthy but nitpick...